src/main/java/it/gov/pagopa/pu/debtpositions/repository/DebtPositionTypeOrgOperatorsRepository.java

```diff
@@ -2,15 +2,14 @@
 
 import it.gov.pagopa.pu.debtpositions.model.DebtPositionTypeOrgOperators;
 import java.util.List;
+import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
-import java.util.Optional;
-
 @RepositoryRestResource(path = "debt-position-type-org-operators")
 public interface DebtPositionTypeOrgOperatorsRepository extends JpaRepository<DebtPositionTypeOrgOperators,Long>{
 
   Optional<DebtPositionTypeOrgOperators> findByDebtPositionTypeOrgIdAndOperatorExternalUserId(Long debtPositionTypeOrgId, String operatorExternalUserId);
   List<DebtPositionTypeOrgOperators> findByDebtPositionTypeOrgId(Long debtPositionTypeOrgId);
-
+  void deleteByDebtPositionTypeOrgId(Long debtPositionTypeOrgId);
```

Comment:
let's return the number of deleted rows in order to log it
```suggestion
  @Transactional
  @Modifying
  long deleteByDebtPositionTypeOrgId(Long debtPositionTypeOrgId);
```